app/views/browse/deployment-config.html

```diff
@@ -129,17 +129,24 @@
                                 <dt>Strategy:</dt>
                                 <dd>{{deploymentConfig.spec.strategy.type}}</dd>
                             </div>
-                            <div ng-if="deploymentConfig.spec.strategy.rollingParams">
-                              <dt>Update Period:</dt>
-                              <dd>{{deploymentConfig.spec.strategy.rollingParams.updatePeriodSeconds}} sec</dd>
-                              <dt>Interval:</dt>
-                              <dd>{{deploymentConfig.spec.strategy.rollingParams.intervalSeconds}} sec</dd>
+
+
+                            <div ng-if="deploymentConfig.spec.strategy.rollingParams || deploymentConfig.spec.strategy.recreateParams">
                               <dt>Timeout:</dt>
-                              <dd>{{deploymentConfig.spec.strategy.rollingParams.timeoutSeconds}} sec</dd>
-                              <dt>Max Unavailable:</dt>
-                              <dd>{{deploymentConfig.spec.strategy.rollingParams.maxUnavailable}}</dd>
-                              <dt>Max Surge:</dt>
-                              <dd>{{deploymentConfig.spec.strategy.rollingParams.maxSurge}}</dd>
+                              <dd>{{deploymentConfig.spec.strategy[strategyParams].timeoutSeconds}} sec</dd>
+                              <div ng-if="deploymentConfig.spec.strategy.rollingParams">
```

Comment:
While you're in the code, can you change this to...?

```html
<dt ng-if-start="deploymentConfig.spec.strategy.rollingParams">Update Period:</dt>
<dd>{{deploymentConfig.spec.strategy.rollingParams.updatePeriodSeconds}} sec</dd>
...
<dt>Max Surge:</dt>
<dd ng-if-end>{{deploymentConfig.spec.strategy.rollingParams.maxSurge}}</dd>
```

Then you don't have the unnecessary div in the middle of the `dl`.